### DISCUSSION TITLE
PRs Mergeable check is not working

### DISCUSSION THREAD

**Discussion by hadiidbouk** [community/community#36542] [Pull Requests]
![Screen Shot 2022-10-18 at 1 34 33 PM](https://user-images.githubusercontent.com/19314956/196407424-a1c064b9-e4ee-4540-90f6-e485701b7a4b.png)

The build is succeeded but Mergeable check still in the pending state

**Comment by ossanna16** [comment]
Hey there @hadiidbouk and thank you for asking a good question! I'm glad you found our community 🙂
